test/test.js

```diff
@@ -437,6 +438,10 @@ describe(`snabbdom-specific rendering`, function() {
     expect(html).to.contain(`bar="baz"`);
   });
 
+  it(`serializes object attributes`, function() {
+    expect(html).to.contain(`obj="${escape(JSON.stringify({hello: `world`}))}"`);
```

Comment:
it's a test case, let's keep it simple and check the actual HTML string
```suggestion
    expect(html).to.contain(`obj="{&quot;hello&quot;:&quot;world&quot;}"`);
```